v4/export/sql.go

```diff
@@ -401,14 +401,14 @@ func estimateCount(dbName, tableName string, db *sql.DB, field string, conf *Con
 	var estRows int
 	if conf.ServerInfo.ServerType == ServerTypeTiDB {
 		/* tidb results field name is estrows
-		+-----------------------+----------+-----------+-----------------------------------------+
-		| id                    | estrows  | task      | operator info                           |
-		+-----------------------+----------+-----------+-----------------------------------------+
-		| tablereader_5         | 10000.00 | root      | data:tablefullscan_4                    |
-		| └─tablefullscan_4     | 10000.00 | cop[tikv] | table:a, keep order:false, stats:pseudo |
-		+-----------------------+----------+-----------+----------------------------------------
+		+-----------------------+----------+-----------+---------------------------------------------------------+
+		| id                    | estrows  | task      | access object | operator info                           |
+		+-----------------------+----------+-----------+---------------------------------------------------------+
+		| tablereader_5         | 10000.00 | root      |               | data:tablefullscan_4                    |
+		| └─tablefullscan_4     | 10000.00 | cop[tikv] | table:a       | table:a, keep order:false, stats:pseudo |
+		+-----------------------+----------+-----------+----------------------------------------------------------
 		*/
-		estRows = detectEstimateRows(db, query, "tidb", 1, 4)
+		estRows = detectEstimateRows(db, query, "tidb", 1)
```

Comment:
perhaps safer to use the column name (`estrows`) rather than column offset (`1`)